### DISCUSSION TITLE
Billable filter

### DISCUSSION THREAD

**Discussion by digilocker** [mcndt/obsidian-toggl-integration#105] [Feature Requests]
Hey @mcndt many thanks for the awesome plugin, have been really impressed with the work you have put into this and the documentation is so clean as well, so apologies if I have missed something here.  Was just wondering if there was a way to filter by billable entries?

**Comment by mcndt** [comment]
Hi Jonny, 

Currently there is no way to filter by billable time entries using TQL. I'll create an issue for adding this.

Note that currently plugin development for this plugin is a bit slow, so apologies if I cannot accommodate your request in time.
